src/pages/Settings/Layout/SettingsMenu/SettingsMenu.tsx

```diff
@@ -24,6 +24,13 @@ const links: Array<MenuItem[]> = [
       icon: '🗝',
     },
   ],
+  [
+    {
```

Comment:
as I undestand this shoudn't be if no signer flag

Reply:
it must be available if there's no keplr extension, it's not desktop only